pkg/cdn/metadata/mongo/mongo.go

```diff
@@ -0,0 +1,40 @@
+package mongo
+
+import "github.com/globalsign/mgo"
+
+// MetadataStore represents a Mongo backed metadata store.
+type MetadataStore struct {
+	session *mgo.Session
+	db      string
+	col     string
+	url     string
+}
+
+// NewMongoStorage returns an unconnected Mongo backed storage
+// that satisfies the Storage interface.  You must call
+// Connect() on the returned store before using it.
+func NewMongoStorage(url, dbName string) *MetadataStore {
+	return &MetadataStore{url: url, db: dbName}
+}
+
+// Connect conntect the the newly created mongo backend.
+func (m *MetadataStore) Connect() error {
+	s, err := mgo.Dial(m.url)
+	if err != nil {
+		return err
+	}
+	m.session = s
+
+	// TODO: collection as env var, or param to New()? together with user/mongo
```

Comment:
I think this is ok as a constant for now. we can make it an env var if we need to, later on